<<<<<<< HEAD:board/laird/wb45n/patches/linux-4.4.39/linux-4.4.39-0046-laird-Enabling-the-BT-config.patch
From 2595998132d0238b4f93708ed4876fa43e552449 Mon Sep 17 00:00:00 2001
From: James Kalb <James.Kalbfleisch@lairdtech.com>
Date: Wed, 31 Aug 2016 16:05:09 -0400
Subject: [PATCH] laird:  Enabling the BT config.
=======
From 8c6ecd7039467f280c511a6d02ee603068ee88ea Mon Sep 17 00:00:00 2001
From: jameskalb <James.Kalbfleisch@lairdtech.com>
Date: Fri, 15 Sep 2017 15:27:16 -0400
Subject: [PATCH 1/1] laird: Enable the usb bluetooth driver
>>>>>>> f2e29e3... laird: modify the patch files to enable the bluetooth drivers:board/laird/wb45n/patches/linux-4.4.39/linux-4.4.39-0049-laird-Enabling-the-BT-config.patch

---
 arch/arm/configs/wb45n_defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/configs/wb45n_defconfig b/arch/arm/configs/wb45n_defconfig
index e3eb628..4127fda 100644
--- a/arch/arm/configs/wb45n_defconfig
+++ b/arch/arm/configs/wb45n_defconfig
@@ -116,6 +116,9 @@ CONFIG_BRIDGE_EBT_LOG=m
 CONFIG_BRIDGE_EBT_NFLOG=m
 CONFIG_BRIDGE=m
 CONFIG_NET_PKTGEN=m
+CONFIG_BT=m
+CONFIG_BT_HCIBTUSB=m
+CONFIG_BT_ATH3K=m
 CONFIG_CFG80211=m
 CONFIG_NL80211_TESTMODE=y
 CONFIG_CFG80211_DEVELOPER_WARNINGS=y
--
2.7.4

